language/stdlib/modules/libra_system.mvir

```diff
@@ -24,7 +24,18 @@ module LibraSystem {
 
     resource BlockMetadata {
       // Height of the current block
+      // TODO: Should we keep the height?
       height: u64,
+
+      // current real world time provided by consensus.
+      timestamp: u64,
+
+      // hash of the previous block
+      id: bytearray,
+
+      // A serialized btreemap of block votes from previous block.
+      previous_block_votes: bytearray,
+      proposer: address,
```

Comment:
Proposer of the previous block, or current one? I think we should be super explicit about this in both the name and the comment.